wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallDegradationViewController/UIAlertController+DegradedCall.swift

```diff
@@ -92,12 +93,36 @@ extension UIAlertController {
                     confirmationBlock(true)
                 })
 
-                controller.addAction(.cancel())
+                controller.addAction(.cancel(cancelBlock))
             } else {
-                controller.addAction(.ok())
+                controller.addAction(.ok({ _ in
+                    cancelBlock?()
+                }))
             }
 
             return controller
         }
 
+    static func incomingCallDegradedMLSConference(
+        confirmationBlock: (@escaping (_ answerDegradedCall: Bool) -> Void)) -> UIAlertController {
```

Comment:
```suggestion
    static func makeIncomingCallDegradedMLSConference(
        confirmationBlock: (@escaping (_ answerDegradedCall: Bool) -> Void)
    ) -> UIAlertController {
```

Two suggestions:
1. add the verb `make` (or `build`) to this func to make clear on the caller side that this is more than a simple property getter.
2. break line after parameter as we use to do the indentation also for multiple parameters, that's it's easy to add something in future and consistent.

Reply:
1. ok
2. it does break searching for function well when you search with text

Reply:
Hmm ok I didn't know that and usually format this way :/